.github/workflows/package.yml

```diff
@@ -18,49 +24,97 @@ jobs:
           with:
             python-version: '3.7'
 
+        - name: Setup conda (windows-latest)
+          if: matrix.os == 'windows-latest'
+          uses: s-weigand/setup-conda@v1
+
+        - name: Setup conda paths (windows-latest)
+          if: matrix.os == 'windows-latest'
+          run: |
+            echo "LIBRARY_DIRS=C:\Miniconda\Library\lib;C:\Miniconda\Library\bin" >> $env:GITHUB_ENV
+            echo "INCLUDE_DIRS=C:\Miniconda\Library\include" >> $env:GITHUB_ENV
+
         - name: Install cibuildwheel
           run: |
             python -m pip install cibuildwheel==2.3.1
-
-        - name: Build wheels for linux python versions
+        - name: Build wheels
           run: |
             python -m cibuildwheel --output-dir dist
           env:
-            CIBW_BUILD: '{cp,pp}3*-many*'
+            CIBW_BUILD: '{cp,pp}3*'
             CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
             CIBW_BEFORE_ALL_LINUX: yum -y install epel-release hdf hdf-devel && ln -s /usr/lib64/hdf/lib* /usr/lib64/
-            CIBW_ARCHS_LINUX: 'x86_64'
-        
-        - name: purge old _linux_wheels
+            CIBW_ARCHS_LINUX: 'x86_64'  # restrict to 64bit builds
+            CIBW_ARCHS_WINDOWS: 'AMD64'  # restrict to 64bit builds
+            # (mac-os) Install hdf-4.2.15 from sources
+            CIBW_BEFORE_ALL_MACOS: >
+              brew install ninja &&
+              cd /tmp &&
+              git clone https://github.com/HDFGroup/hdf4.git &&
+              mkdir build && cd build &&
+              ../hdf4/configure --enable-shared --disable-static --disable-fortran --disable-netcdf --enable-java --enable-production --with-zlib --prefix=/usr/local &&
```

Comment:
why do we need `--enable-java`?

Reply:
We don't! I was lazy and just used the default configure parameters, sorry!